test/unit/sync/commits.test.ts

```diff
@@ -1,237 +1,245 @@
 /* eslint-disable @typescript-eslint/no-var-requires,@typescript-eslint/no-explicit-any */
-import { commitsNoLastCursor, commitsWithLastCursor, getDefaultBranch } from "../../fixtures/api/graphql/commit-queries";
```

Comment:
Please take extra care looking through these tests. They were previously all skipped, have rebuilt them all based off the original test cases.